requirements.txt

```diff
@@ -47,21 +51,21 @@ pycparser==2.14
 pycrypto==2.6.1
 pyOpenSSL==0.15.1
 pyrsistent==0.11.9
-python-cinderclient==1.1.1
+python-cinderclient==1.6.0
```

Comment:
This is the important update. 
It turns out that  `python-cinderclient==1.1.1` didn't make any use of the `next` paging links returned in Cinder V2 response body. 
Upgrading `python-cinderclient` required an update to `python-novaclient` and a bunch of transient dependencies too.
This seems to be the minimum required to make pip / setuptools happy.